server/controllers/request.js

```diff
@@ -0,0 +1,83 @@
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const Request = require("../models/Request");
+
+exports.createRequest = async (req, res) => {
+    const { ownerId, sitterId, start, end, animalType, usefulInfo } = req.body;
+    console.log(new Date());
+    try {
+        const ownerExists = await User.findOne({ ownerId });
+        if (!ownerExists) return res.status(404).json({ message: "Owner doesn't exist" });
+
+        const sitterExists = await User.findOne({ sitterId });
+        if (!sitterExists) return res.status(404).json({ message: "Sitter doesn't exist" });
+
+        const request = await Request.create(req.body);
+        if (!request) return res.status(404).json({ message: "unable to create a new request" });
+
+        res.status(200).json(request);
+    } catch (error) {
+        res.status(404).json({ message: error.message });
```

Comment:
```suggestion
        res.status(500).json({ message: error.message });
```
A `500` seems more appropriate here